vendor/k8s.io/kubernetes/pkg/kubelet/kubelet_server_journal.go

```diff
@@ -76,6 +77,10 @@ func newJournalArgsFromURL(query url.Values) (*journalArgs, error) {
 	if err != nil {
 		return nil, fmt.Errorf("parameter 'unit' is invalid: %v", err)
 	}
+	boot, err := safeString(query.Get("boot"))
```

Comment:
Add a test case to verify it gets propagated and is safe too.  Also, I would prefer boot only support numbers in the range 0 > -100.  We have to very carefully sanitize input because we could potentially open root level exposure on the host.